app/assets/stylesheets/components/internal_link.scss

```diff
@@ -12,4 +12,22 @@
     background-repeat: no-repeat;
     background-position: 0 .25em;
   }
+}
+
+// Keep from being hidden behind nav bar on small screens.
+@media (max-width: 992px) {
+  .course-section__title::before, .lesson-content h3::before {
+    display: block; 
+    content: " "; 
+    margin-top: -100px; 
+    height: 100px; 
+    visibility: hidden; 
+    pointer-events: none;
+  }
+}
+
+@media (max-width: 600px){
```

Comment:
Can this use be moved within the scope of `.internal-link` please? and can we use the media mixin from bootstrap for this instead?
```
.internal-link {
  ...

  @include media-breakpoint-down(sm) {
    background-image: none;
  }
}
```